src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(array_methods)]
+#![feature(map_into_keys_values)]
```

Comment:
Not sure what is going on here. Locally when I run `make check` it says I do not need this because the feature is stable. But CI seems to think it is needed, and even when I add it here [it still fails](https://github.com/aurora-is-near/aurora-engine/pull/227/checks?check_run_id=3314455527).

Reply:
`rustup update` it was recently made stable. If CI is failing then it is building with an older rust v.

Reply:
Ok, @strokovok and I worked together to resolve this.